packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,287 @@
+                        multiple={false}
+                        required={false}
+                        placeholder={''}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class AssignedToGroupInput extends React.PureComponent<SecurityUserGroupProps, any> {
+
+    getHelpTip() {
+        return ISSUES_LIST_GROUP_ASSIGN_TIP;
+    }
+
+    onChange = (name: string, formValue: any, selected: Principal, ref: any): any => {
+        if (selected && this.props.onSelect) {
+            this.props.onSelect(selected, name);
+        }
+    };
+
+    render() {
+        const { model, coreGroups } = this.props;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Assigned to list comes from" helpTipBody={this.getHelpTip} required={false} />
+                </Col>
+                <Col xs={9} lg={8}>
+                    <SelectInput
+                        name={'assignedToGroup'}
+                        options={coreGroups.toArray()}
+                        placeholder=""
+                        inputClass="col-xs-12"
+                        valueKey="userId"
+                        labelKey="displayName"
+                        onChange={this.onChange}
+                        value={model.assignedToGroup ? model.assignedToGroup : undefined}
+                        formsy={false}
+                        showLabel={false}
+                        multiple={false}
+                        required={false}
+                    />
+                </Col>
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class DefaultUserAssignmentInput extends React.PureComponent<SecurityUserGroupProps, any> {
+
+    getHelpTip() {
+        return ISSUES_LIST_USER_ASSIGN_TIP;
+    }
+
+    onChange = (name: string, formValue: any, selected: Principal, ref: any): any => {
+        if (selected && this.props.onSelect) {
+            this.props.onSelect(selected, name);
+        }
+    };
+
+    getFilteredCoreUsers = (groupId: any, coreUsers: List<UserGroup>) => {
+        let filteredCoreUser = coreUsers.filter(coreUser => {
+            return coreUser.groupId === groupId;
+        });
+        return filteredCoreUser.toArray().length > 0 ? filteredCoreUser.toArray() : undefined;
+    };
+
+    render() {
+        const { model, coreUsers } = this.props;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Default user assignment" helpTipBody={this.getHelpTip} required={false} />
+                </Col>
+                <Col xs={9} lg={8}>
+                    <SelectInput
+                        name={'assignedToUser'}
+                        options={this.getFilteredCoreUsers(model.assignedToGroup, coreUsers)}
+                        placeholder=""
```

Comment:
Similar to the comment above, I would use this placeholder text as a way to tell the user what the blank option value would be (i.e. "No default")

Reply:
done